components/Footer.js

```diff
@@ -1,7 +1,95 @@
+import styles from '../styles/Footer.module.css';
+import Link from 'next/link'
+
 export default function Footer() {
     return (
-        <footer>
-           Footer
+        <footer className={styles.footer}>
+
+            <div className={styles.topLine}>
+                <div className={styles.logoBox}>
+                    <Link href="/"><img src="/HMLogo.svg" alt="logo" className={styles.logo} /></Link>
+
+                </div>
+
+                <div className={styles.leftBox}>
+
+                    <img src="/mapicon.svg" className={styles.mapIcon} />
+
+                    <div className={styles.addressBox}>
+
+                        <p>8171 Main St</p>
```

Comment:
We should wrap this whole address section in a `Link` that links to the Google Maps address of the store https://goo.gl/maps/eAUVuMgsLMhacw3S7